homeassistant/components/sleepiq/manifest.json

```diff
@@ -1,9 +1,10 @@
 {
   "domain": "sleepiq",
   "name": "SleepIQ",
+  "config_flow": true,
   "documentation": "https://www.home-assistant.io/integrations/sleepiq",
   "requirements": ["sleepyq==0.8.1"],
-  "codeowners": [],
+  "codeowners": ["@mfugate1"],
   "iot_class": "cloud_polling",
```

Comment:
https://ouilookup.com/vendor/sleep-number

Assuming these are the corrupt OUIs for the devices and they use DHCP to get an ip address on the network you can trigger the config flow to start a flow automatically when that OUI+hostname is seen on the network

https://developers.home-assistant.io/docs/creating_integration_manifest?_highlight=dhc#dhcp

If you can't find the hostname/OUI for the device, enable debug logging for `homeassistant.components.dhcp` and power cycle the device so it makes a new dhcp request

Reply:
The OUI for my bed ended up being a vendor called "Select Comfort." I can't confirm whether the sleep-number vendor is correct, so I didn't add those two OUIs. 

The hostname for mine started with 500-. It's unclear whether that will change based on bed model, so I did not specify any hostname.

Reply:
Sounds good. Worst case we have to add a stricter version later if we get a false positive (can always be ignored though)